### PR TITLE
Solution #02

### PR DESCRIPTION
**Pull request by tuommii**
Checking if values are valid and then converting those to hex with sprintf-function. No magic numbers.

### CODE REVIEW

File: chall02/mtuomine.c [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#define ARR_LEN 3
+#define HEX_LEN 7
+#define MIN_VALUE 0
+#define MAX_VALUE 255
```
Comment (jgengo) [inline chall02/mtuomine.c:7]: why not :) a bit overkill for such a small project.

File: chall02/mtuomine.c [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	int arr[ARR_LEN] = {r, g, b};
+
+	if (!is_valid(arr))
+		return NULL;
+
+	char *hex = (char *)malloc((sizeof(char) * HEX_LEN) + 1);
+	if (!hex)
+		return NULL;
+
+	sprintf(hex, "%s%02x%02x%02x", "#", r, g, b);
```
Comment (jgengo) [inline chall02/mtuomine.c:32]:
```c
sprintf(hex, "#%02x%02x%02x, r, g, b);
```

why did you add this const as an arg?

File: chall02/mtuomine.c [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	hex[HEX_LEN] = '\0';
```
Comment (jgengo) [inline chall02/mtuomine.c:33]: unnecessary, sprintf already add a trailing null

**Review by jgengo — COMMENTED** [review]
good job!

File: chall02/mtuomine.c (hunk 2 above)
Comment (tuommii) [reply chall02/mtuomine.c:32]: Easier to replace it or make it dynamic in the future, So my asnwer is maintainability 😂
